pombola/south_africa/static/sass/_south-africa_overrides.scss

```diff
@@ -1173,6 +1173,9 @@ $logo-width-large: 184px;
                     line-height: 1.4;
                     letter-spacing: 0.01em;
                     width: 95%;
+                    @media screen and (max-width: 479px) {
+                        max-width: 80%;
+                    }
```

Comment:
I don't think it should apply to all buttons - only buttons with icons on the right.

Reply:
noted, declining PR. I'll create another one.